### ISSUE TITLE
GUI implementation

### ISSUE THREAD

**Issue by Firedrops** [Firedrops/qimr_mtb#1]
Discussion here

https://blog.jessfraz.com/post/docker-containers-on-the-desktop/

Suggested parameters:
```
    -v /tmp/.X11-unix:/tmp/.X11-unix \ # mount the X11 socket
    -e DISPLAY=unix$DISPLAY \ # pass the display
    --device /dev/snd \ # sound
```

to be tested

**Comment by Firedrops** [comment]
an implementation by beast2's authors found here. Uses vnc locally to send through the GUI.
https://github.com/CompEvol/beast2/blob/master/Dockerfile

**Comment by Firedrops** [comment]
Finally resolved with e5f6c792ab28dcb33ed4af9f2424747386b4af5c.

Installed XFCE4, and GUI passed through the VNC protocol.
